### ISSUE TITLE
Interface design: line

### ISSUE THREAD

**Issue by noyannus** [program-in-chinese/HistoryInThreads_WebExtension#20]
Please add an option to set the font (and size, please, please, please!) for the history threads.

Currently, line height is too small for the hardcoded font for western fonts, see image. 

(Also, there is appears an ugly dark bar under the filter/search panel on the top, at least with some themes / FF versions / OSes)

Firefox 66.0.5
macOS 10.14.5

![Screenshot 2019-05-22 um 10 28 05](https://user-images.githubusercontent.com/6339921/58159524-02b9d480-7c7d-11e9-9ca2-8d59bb14fddf.png)

**Comment by nobodxbodon** [comment]
Thanks for the report. May I ask if you used any non-default font setting in Firefox? Because I don't find a way to reproduce this issue on my Mac 10.13.6.

If you mean this shade part below, I can try to make it less obvious.
![Screen Shot 2019-05-24 at 10 23 45 AM](https://user-images.githubusercontent.com/392497/58345809-21030a00-7e0e-11e9-913f-cc26b77bdbc8.png)

**Comment by noyannus** [comment]
Yes, I have set font sizes in Firefox. (I also use Zoom Page WE, but that seems to leave the sidebar alone.)

Setting here...
![Screenshot 2019-05-24 um 23 41 51](https://user-images.githubusercontent.com/6339921/58358139-f2247c80-7e7d-11e9-835b-b98b565c3e30.png)

to mimimum font size of 20px look like this:

![Screenshot 2019-05-24 um 23 41 37](https://user-images.githubusercontent.com/6339921/58358156-05374c80-7e7e-11e9-8047-4287031d3452.png)

And to 14px like this:

![Screenshot 2019-05-24 um 23 40 50](https://user-images.githubusercontent.com/6339921/58358168-141dff00-7e7e-11e9-9cea-cd836b428680.png)

So maybe the line height does not get updated along with the font size? Still, a user customizable setting would make HiT settings independent from the overall FF settings. Some vision-impaired folks (and the tweakers, like me) would surely like that.

===========

The shadow is too strong, IMO, on light themes like Light (or is it Bright in English?), Chrome Grey, Solid Grey, etc.. With dark themes it looks better. As the rest of FF windows doesn't use shadows, you could probably just omit it (but that is a matter of personal taste, not really an issue!).

===========

I stumbled on something else while testing font sizes: The Search button vanishes when the sidebar is wide.

![Screenshot 2019-05-24 um 23 34 01](https://user-images.githubusercontent.com/6339921/58358450-3d8b5a80-7e7f-11e9-8626-97db8c75c3b1.png)

![Screenshot 2019-05-24 um 23 34 12](https://user-images.githubusercontent.com/6339921/58358455-411ee180-7e7f-11e9-941d-578a7bf9682a.png)

If the button is meant to be on a line with the filter menu (Today, Yesterday,...), then it would be good to  make it narrower (a square with a looking glass or filter icon, maybe? And on the right side of the text field?). This would allow more of the search string to be visible in the search field (or to better read it at a bigger font size). 


The menu has no indicator that it is a menu. How about adding a triangle like in the FF dialog in the image above?

===========
Finally, thanks for making this Add-On. It has already become quite useful for me!
